src/Microsoft.Data.SqlClient/tests/UnitTests/IO/TdsWriteStreamTest.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Threading;
+using Microsoft.Data.SqlClientX.IO;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.UnitTests.IO
```

Comment:
Can fix namespace to 'UnitTests.IO' ?

Reply:
Ya

Reply:
@cheenamalhotra  looking at the tests in FunctionalTests and Manual tests, we are following the fully qualified namespace where we are prefixing the NS with Microsoft.Data.SqlClient. I would like to stick to the pattern.